libs/net/ches/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -24,5 +24,20 @@ public static IServiceCollection AddChesService(this IServiceCollection services
                 .AddScoped<IHttpRequestClient, HttpRequestClient>()
                 .AddTransient<JwtSecurityTokenHandler>();
         }
+
+        /// <summary>
+        /// Add the AddChesService to the dependency injection service collection.
+        /// </summary>
+        /// <param name="services"></param>
+        /// <param name="section"></param>
+        /// <returns></returns>
+        public static IServiceCollection AddChesSingletonService(this IServiceCollection services, IConfigurationSection section)
```

Comment:
This is required for the services we build.  I'd like to figure out how to do scoped at some point.